Gems/PhysX/Code/Source/PrismaticJointComponent.h

```diff
@@ -27,9 +29,24 @@ namespace PhysX
         PrismaticJointComponent(
             const JointComponentConfiguration& configuration,
             const JointGenericProperties& genericProperties,
-            const JointLimitProperties& limitProperties);
+            const JointLimitProperties& limitProperties,
+            const JointMotorProperties& motorProperties);
         ~PrismaticJointComponent() = default;
 
+        // JointInterfaceRequestBus::Handler
+        float GetPosition() override;
+        float GetVelocity() override;
+        AZ::Transform GetTransform() override;
+
+        // JointMotorRequestBus ::Handler
+        void SetVelocity(float velocity) override;
+        void SetMaximumForce(float force) override;
```

Comment:
update: API comment block seems incorrect here again (should just be `JointInterfaceRequestBus overrides ...`)

Reply:
Adjusted to block comments here.